web/themes/custom/subtheme_videogame/src/sass/base/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+$orange: #F57C00;
```

Comment:
Please place the colors in a separate `_colors.scss` file and import it before the import of `_variables.scss`.